apps/api/src/app/rate-limiting/usecases/get-api-rate-limit-maximum/get-api-rate-limit-maximum.command.ts

```diff
@@ -0,0 +1,9 @@
+import { IsDefined, IsEnum } from 'class-validator';
+import { ApiRateLimitCategoryEnum } from '@novu/shared';
+import { EnvironmentCommand } from '../../../shared/commands/project.command';
+
+export class GetApiRateLimitMaximumCommand extends EnvironmentCommand {
+  @IsDefined()
+  @IsEnum(ApiRateLimitCategoryEnum)
+  apiRateLimitCategory: ApiRateLimitCategoryEnum;
+}
```

Comment:
This was a refactor from the following file (below in PR) that wasn't picked up correctly by git:
https://github.com/novuhq/novu/pull/4867/files#diff-bb13833ec972ae0f2f3cd90b6ac0c9f68be590818abe737ca3f26baf85516740